bundles/pubsub/pubsub_spi/src/pubsub_utils.c

```diff
@@ -143,12 +143,17 @@ celix_properties_t *pubsub_utils_getTopicProperties(const celix_bundle_t *bundle
         bundle_getEntry((celix_bundle_t *)bundle, ".", &bundleRoot);
 
         if (bundleRoot != NULL) {
-            asprintf(&topicPropertiesPath, "%s/META-INF/topics/%s/%s.properties", bundleRoot, isPublisher? "pub":"sub", topic);
+            asprintf(&topicPropertiesPath, "%s/META-INF/topics/%s/%s.%s,properties", bundleRoot, isPublisher? "pub":"sub", topic, scope);
```

Comment:
Same here, I would expect the scope to be before the topic, as it also is in the discovery entry.

Also, typo in the name, "...%s.%s,properties", that comma should be a point.

Reply:
I have fixed this